### PR TITLE
Component should update on style change. Fixes #38

### LINKED ISSUE #38: Component should update on style change.
In response to the suggested change here: https://github.com/lovasoa/react-contenteditable/pull/36

Currently prop style changes are not rendered by the component.

I had a go at reflecting these changes for style but I believe it would require a deep equality comparison on the style prop passed in to understand that a render needs to occur.

You can see the changes I made as a temporary measure here: https://github.com/jimah/react-contenteditable/blob/master/src/react-contenteditable.js#L26

Lodash has a nice deep equal function but that would be adding a dependency.

### PR DESCRIPTION
**Pull request by ajbeaven**
Thanks to @monolithed for the code in #36.

### CODE REVIEW

File: src/react-contenteditable.js [hunk 1]
```diff
@@ -23,19 +23,25 @@ export default class ContentEditable extends React.Component {
...
+    if (!htmlEl) {
+      return true;
+    }
+
+    // ...or if html really changed... (programmatically, not by user edit)
+    if (nextProps.html !== htmlEl.innerHTML && nextProps.html !== props.html) {
+      return true;
+    }
+
+    let optional = ['style', 'className', 'disable'];
```
Comment (ryanscottaudio) [inline src/react-contenteditable.js]: what about the `tagName` prop?

Comment (ajbeaven) [reply src/react-contenteditable.js]: Ok, added - though it's a bit redundant considering this repo seems to have been abandoned.

Comment (ryanscottaudio) [reply src/react-contenteditable.js]: hahaha that's fair

**Comment by lovasoa** [comment]
Can you update the compiled file too ? I'm going to merge this.

**Comment by ajbeaven** [comment]
Ok, done.

**Comment by Goues** [comment]
Hi, this PR is just what I'm looking for, can you push a new release of this package to NPM, so that I can use this? Thank you very much.

**Comment by lovasoa** [comment]
@Goues [Done](https://www.npmjs.com/package/react-contenteditable).

**Pull request merged**